### PR TITLE
chore: remove unused dependencies

### PR DESCRIPTION
**Pull request by mcmxcdev**
## Changes
- uninstall various dependencies which are not in use

## Motivation
Slim down amount of dependencies to manage

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,22 +98,18 @@
...
 		"autoprefixer": "^10.4.14",
 		"clsx": "^1.2.1",
-		"escape-html": "^1.0.3",
 		"eslint": "^8.45.0",
 		"eslint-config-prettier": "^8.9.0",
 		"eslint-plugin-storybook": "^0.6.15",
 		"eslint-plugin-svelte": "^2.33.0",
 		"fathom-client": "^3.5.0",
 		"glob": "^9.3.2",
-		"happy-dom": "^10.5.2",
```
Comment (mcmxcdev) [inline package.json:120]: Vitest uses jsdom already in vite config, so happy-dom is not needed.

File: package.json [hunk 2]
```diff
@@ -102,22 +98,18 @@
 		"@testing-library/svelte": "^4.0.3",
 		"@testing-library/user-event": "^14.4.3",
 		"@types/jest-axe": "^3.5.5",
-		"@types/prettier": "^2.7.3",
-		"@types/testing-library__jest-dom": "^5.14.9",
```
Comment (mcmxcdev) [inline package.json:106]: These are stub type definitions, meaning that the dependencies themselves are already providing type definitions nowadays.

File: package.json [hunk 3]
```diff
@@ -84,14 +84,10 @@
 		"@storybook/addon-interactions": "^7.6.6",
 		"@storybook/addon-links": "^7.6.6",
 		"@storybook/addon-styling": "^1.3.7",
-		"@storybook/blocks": "^7.6.6",
 		"@storybook/svelte": "^7.6.6",
 		"@storybook/sveltekit": "^7.6.6",
 		"@storybook/testing-library": "^0.2.2",
-		"@sveltejs/adapter-auto": "^2.1.0",
-		"@sveltejs/adapter-cloudflare": "^2.3.4",
```
Comment (mcmxcdev) [inline package.json:92]: `@sveltejs/adapter-static` is the only adapter that is in use.

**Comment by TGlide** [comment]
Closing since I removed some more. Thank you for this though, it helped a ton!

**Pull request closed without merging**